Now getting errors like

```
numpy/core/src/umath/simd.inc.src: In function ‘fma_scalef_ps’:
  numpy/core/src/umath/simd.inc.src:1519:1: warning: AVX vector return without AVX enabled changes the ABI [-Wpsabi]
   {
   ^
  numpy/core/src/umath/simd.inc.src:1518:1: note: The ABI for passing parameters with 32-byte alignment has changed in GCC 4.6
   fma_scalef_ps(__m256 poly, __m256 quadrant)
   ^~~~~~~~~~~~~
  In file included from /usr/lib/gcc/x86_64-linux-gnu/7/include/immintrin.h:41:0,
                   from numpy/core/src/umath/simd.inc.src:25,
                   from numpy/core/src/umath/loops.c.src:50:
  /usr/lib/gcc/x86_64-linux-gnu/7/include/avxintrin.h:1195:1: error: inlining failed in call to always_inline ‘_mm256_movemask_ps’: target specific option mismatch
   _mm256_movemask_ps (__m256 __A)
   ^~~~~~~~~~~~~~~~~~
```

I was getting this locally as well but it wasn't clear at first that this was related.  Seems maybe it is.

--- a/numpy/distutils/command/autodist.py
+++ b/numpy/distutils/command/autodist.py
@@ -69,7 +69,10 @@ def check_gcc_function_attribute(cmd, attribute, name):
         #pragma GCC diagnostic error "-Wattributes"
         #pragma clang diagnostic error "-Wattributes"
 
-        int %s %s(void*);
+        int %s %s(void* unused)
+        {
+            return 0;
+        }
 
         int
         main()

--- a/test_requirements.txt
+++ b/test_requirements.txt
@@ -1,5 +1,5 @@
 cython==0.29.17
-hypothesis==5.12.0
+hypothesis==5.14.0
 pytest==5.4.2
 pytz==2020.1
 pytest-cov==2.8.1

